packages/package_info/README.md

```diff
@@ -1,19 +1,21 @@
-# package_info
+# PackageInfo
 
 This Flutter plugin provides an API for querying information about an
 application package.
 
 # Usage
 
-You can use the package_info package to query information about the
+You can use the PackageInfo to query information about the
 application package. This works both on iOS and Android.
 
 ```dart
-import 'package:package_info/package_info.dart' as package_info;
+import 'package:package_info/package_info.dart';
 
-var version = await package_info.version;
-var buildNumber = await package_info.buildNumber;
-var packageName = await package_info.packageName;
+PackageInfo packageInfo = await PackageInfo.getInstance();
+
+String version = packageInfo.version;
+String buildNumber = packageInfo.buildNumber;
+String packageName = packageInfo.packageName;
```

Comment:
Here and everywhere below: Seems to me the `packageName` is now logically the main piece of information (even if it was added most recently). So we should reorder every place these three occur together, including constructor, property declarations and so on below.